src/components/NewLocationPage/ChartFooter/utils.ts

```diff
@@ -20,7 +22,21 @@ export function getAddedMetricStatusText(
   metric: ExploreMetric,
   value: string,
   region: Region,
+  projections: Projections,
 ) {
+  // For county-level hospital data we display copy for the default disaggregated county data as well as
+  // HSA level data. We need to plug in custom copy and re-aggregate the HSA data from the estimated county data.
+  if (
```

Comment:
This feels a little hacky, but I'm not immediately sure of a better way to handle this